release_notes_clear.py

```diff
@@ -0,0 +1,117 @@
+# remove all releaseNotes from files in: Itegrations, Playbooks, Reports and Scripts.
+# Note: using yaml will destroy the file structures so filtering as regular text-file.
+import os
```

Comment:
maybe mark that to use the script, you should be in the root directory & run `python release_notes_clear.py`